compat-data.schema.json

```diff
@@ -1,72 +1,133 @@
         "notes": {
-          "type": "array",
-          "items": {
-            "type": "string"
-          }
+          "anyOf": [
+            {
+              "type": "string"
+            },
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          ]
         }
       },
-      "required": ["support"],
+      "required": ["version_added"],
       "additionalProperties": false
     },
 
+    "array_support_statement": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/simple_support_statement"
+      }
+    },
+
+    "support_statement": {
+      "anyOf": [
+        { "$ref": "#/definitions/simple_support_statement" },
+        { "$ref": "#/definitions/array_support_statement" }
+      ]
+    },
+
     "status_statement": {
       "type": "object",
       "properties": {
         "experimental": { "type": ["string", "boolean"] },
-        "standardized": { "type": ["string", "boolean"] },
-        "stable": { "type": ["string", "boolean"] },
+        "standard_track": { "type": ["string", "boolean"] },
         "obsolete": { "type": ["string", "boolean"] }
       },
-      "required": ["experimental", "standardized", "stable", "obsolete"],
+      "required": ["experimental", "standard_track", "obsolete"],
       "additionalProperties": false
     },
 
-    "feature": {
+    "subfeature": {
       "type": "object",
       "properties": {
-        "Android": { "$ref": "#/definitions/support_statement" },
-        "Android Webview": { "$ref": "#/definitions/support_statement" },
-        "Chrome": { "$ref": "#/definitions/support_statement" },
-        "Chrome for Android": { "$ref": "#/definitions/support_statement" },
-        "Edge": { "$ref": "#/definitions/support_statement" },
-        "Edge Mobile": { "$ref": "#/definitions/support_statement" },
-        "Firefox": { "$ref": "#/definitions/support_statement" },
-        "Firefox for Android": { "$ref": "#/definitions/support_statement" },
-        "IE Mobile": { "$ref": "#/definitions/support_statement" },
-        "Internet Explorer": { "$ref": "#/definitions/support_statement" },
-        "Node.js": { "$ref": "#/definitions/support_statement" },
-        "Opera": { "$ref": "#/definitions/support_statement" },
-        "Opera Mobile": { "$ref": "#/definitions/support_statement" },
-        "Safari": { "$ref": "#/definitions/support_statement" },
-        "Safari Mobile": { "$ref": "#/definitions/support_statement" },
-        "Servo": { "$ref": "#/definitions/support_statement" },
+        "desc": { "type": ["string"] },
+        "support": { "$ref": "#/definitions/compat_block" },
         "status": { "$ref": "#/definitions/status_statement" }
       },
-      "required": ["Chrome", "Opera", "Edge", "Firefox", "Firefox for Android"],
       "additionalProperties": false
     },
 
-    "feature_set": {
+    "compat_block": {
```

Comment:
Very much a nit, but it helps readability if you swap this and "subfeature", so I can read it all the way from the bottom up, going from a reference to its definition.